### ISSUE TITLE
FR wifi esp support for duet3 6hc

### ISSUE THREAD

**Issue by Lino77** [Duet3D/RepRapFirmware#521]
if a rpi is running on the duet3 6hc why is the esp not supported, i have looked at the circuit diagrams the board actually provides everything that hardware is moderately required for it,

**Comment by dc42** [comment]
The MB6HC does not have a ESP module.

**Comment by Lino77** [comment]
I know that he doesn't have one. but you could connect one to the Rpi port and take RX TX from IO_0 or IO_1 and from the other IO ports, or I think wrong

**Comment by romalb47** [comment]
Or use an rpi as wifi controller, it's not really more costly and add a lot's of advantage. Like a VPN or tactil screen for administration. Adding an ESP to the board is redundant and add code that load the MCU and must be supported.

**Comment by dc42** [comment]
Agreed, if you are using an attached RPi then it is much better to use the WiFi provided by the RPi.
